src/app/shared/services/auth.service.ts

```diff
@@ -36,9 +36,9 @@ export interface Capabilities {
   entity: '',
 })
 export class AuthService extends BaseBackendService<BaseModel> {
-  public loggedIn: BehaviorSubject<boolean>;
+  public loggedIn = new BehaviorSubject<boolean>(false);
   // tslint:disable-next-line:variable-name
-  private _user: User | null;
+  public userSubject = new BehaviorSubject<User>(null);
```

Comment:
Add null to type because in getter user you have both. Or remove null type in getter